recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
 {% set name = "scikit-build-core" %}
-{% set version = "0.6.1" %}
+{% set version = "0.10.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit_build_core-{{ version }}.tar.gz
-  sha256: 392254a4ca7235c27a4be98cc24cd708f563171961ce37cff66120ebfda20b7a
+  sha256: b256805c75accc9f7ecd2d94e85d5e470b1cc325612978b7480b277da3ab34ff
 
 build:
-  number: 1
+  number: 0
   skip: true  # [py<37]
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - hatch-vcs
     - hatchling
     - pip
     - python
   run:
     - python
-    - exceptiongroup             # [py<311]
-    - importlib-metadata         # [py<38]
-    - importlib_resources >=1.3  # [py<39]
-    - packaging >=20.9
-    - tomli >=1.1                # [py<311]
-    - typing-extensions >=3.10   # [py<38]
+    - exceptiongroup >=1.0              # [py<311]
+    - importlib-metadata >=4.13         # [py<38]
+    - importlib_resources >=1.3         # [py<39]
+    - packaging >=21.3
+    - tomli >=1.2.2                     # [py<311]
+    - typing-extensions >=3.10.0        # [py<39]
     # pathspec and pyproject-metadata are actually required because
     # injected at runtime:
     # https://github.com/scikit-build/scikit-build-core/blob/d7ba004/src/scikit_build_core/build/__init__.py#L113
```

Comment:
You can remove `pyproject-metadata` since it's now vendored. You can also remove this comment since `pathspec` is now a required dependency upstream too, see https://inspector.pypi.io/project/scikit-build-core/0.10.1/packages/df/1a/6a3d2b5d2ce1b87dbc9d58592cf67741f97eafee68e8008c2f56f8fc1be6/scikit_build_core-0.10.1-py3-none-any.whl/scikit_build_core-0.10.1.dist-info/METADATA#line.35.

Reply:
Removed!